### PR TITLE
fix ./bin/config command not found

### PR DESCRIPTION
**Pull request by cguentherTUChemnitz**
Sorry, i broke up the bootup, by inserting the stuff using ./bin/config below popd.

### CODE REVIEW

File: 10-boot-conf [hunk 1]
```diff
@@ -235,10 +235,9 @@ fi
 sudo -u "$PHABRICATOR_VCS_USER" ./bin/config set notification.servers '[{"type": "client","host": "'$PHABRICATOR_HOST'","port": '$APHLICT_PORT',"protocol": "'$APHLICT_PROTOCOL'","path": "/ws/"},{"type": "admin","host": "127.0.0.1","port": 22281,"protocol": "http"}]'
 unset APHLICT_PROTOCOL
 unset APHLICT_PORT
-popd
-
-#add necessary path to find git-http-backend
+add necessary path to find git-http-backend
```
Comment (hach-que) [inline 10-boot-conf]: Missing `#`?

**Comment by cguentherTUChemnitz** [comment]
updated as requested

**Comment by cguentherTUChemnitz** [comment]
As the #51  is reverted, the image boots up correctly as before. But the #31 is now back. So this pull request can be seen as a corrected version of #51 .

**Comment by hach-que** [comment]
You have conflicts, so I can't squash and merge this branch.

**Comment by huangjimmy** [comment]
Ho @hach-que , has #31 been fixed? It seems that #52 is approved, but I search this repository and I cannot find any code that adds /usr/lib/git to append paths.

**Pull request closed without merging**